packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config.vue

```diff
@@ -691,8 +692,15 @@ const initialize = async () => {
 	const state = props.node.state;
 	const modelId = props.node.inputs[0].value?.[0];
 	if (!modelId) return;
-	fetchConfigurations(modelId);
+	await fetchConfigurations(modelId);
+
 	model.value = await getModel(modelId);
+	if (suggestedConfigurationContext.value.tableData.length === 0 && model.value) {
+		addDefaultConfiguration(model.value).then(() => {
+			fetchConfigurations(modelId);
+		});
+	}
```

Comment:
Please mark the section(s) clearly if the solution is meant to be temporary, so other readers would know this is, and whether or not to build on it.